tests/cephfs/cephfs_bugs/prevent_scrub_with_standby_mds.py

```diff
@@ -0,0 +1,103 @@
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        tc = "62537"
+        log.info(f"Running CephFS tests for ceph tracker - {tc}")
+        # Initialize the utility class for CephFS
+        fs_util = FsUtils(ceph_cluster)
+        # Get the client nodes
+        clients = ceph_cluster.get_ceph_objects("client")
+        config = kw.get("config")
+        # Authenticate the clients
+        fs_util.auth_list(clients)
+        build = config.get("build", config.get("rhbuild"))
+        # Prepare the clients
+        fs_util.prepare_clients(clients, build)
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        # Check if jq package is installed, install if not
+        jq_check = client1.exec_command(
+            sudo=True, cmd="rpm -qa | grep jq", check_ec=False
+        )
+        if "jq" not in jq_check:
+            client1.exec_command(sudo=True, cmd="yum install -y jq")
+        # Generate random string for directory names
+        rand = "".join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(5)
+        )
+        # Define mount directories
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse_{rand}"
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel_{rand}"
+        # Mount CephFS using ceph-fuse and kernel
+        fs_util.fuse_mount([client1], fuse_mounting_dir_1)
+        mon_node_ips = fs_util.get_mon_node_ips()
+        fs_util.kernel_mount([client1], kernel_mounting_dir_1, ",".join(mon_node_ips))
+        # create directories in the mounted directory
+        num_dir = 100
+        for i in range(num_dir):
+            client1.exec_command(
+                sudo=True, cmd=f"mkdir {fuse_mounting_dir_1}/dir_fuse_{i}"
+            )
+            client1.exec_command(
+                sudo=True, cmd=f"mkdir {kernel_mounting_dir_1}/dir_kernel_{i}"
+            )
+        # set max_mad to 1
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs max_mds 1")
+        # set allow standby mds to true
+        client1.exec_command(
+            sudo=True, cmd="ceph fs set cephfs allow_standby_replay true"
+        )
+        # wait until standby mds is created
+        standby_mds_cmd = (
+            "ceph fs status cephfs -f json-pretty | jq -r '.mdsmap[] "
+            '| select(.state == "standby-replay") | .name\''
+        )
+        standby_mds, _ = client1.exec_command(sudo=True, cmd=standby_mds_cmd)
+        standby_mds = standby_mds.rstrip()
+        # run scrub with standby mds
+        log.info(f"standby mds:{standby_mds}")
+        scrub_cmd = f"ceph tell mds.{standby_mds} scrub start /"
+        scrub_output, _ = client1.exec_command(sudo=True, cmd=scrub_cmd, check_ec=False)
+        log.info(f"scrub result:{scrub_output}")
+        if '"return_code": 0' in scrub_output:
+            log.error("With standby mds, scrub should not run")
```

Comment:
Please add a comment in test suite file for an expected failure due to open BZ https://bugzilla.redhat.com/show_bug.cgi?id=2294478 , when you plan to add automation to suite file. Are you not planning to include it until Z is fixed?

Reply:
I have added the BZ.

 Are you not planning to include it until Z is fixed? -> I was planning adding it after unitl Z is fixed. is it okay to add now?